build/configs/artik053/artik053_download.sh

```diff
@@ -64,15 +67,44 @@ main()
 				exit 1
 			fi
 
+			# Prepare for ROMFS
+			if [ "${CONFIG_FS_ROMFS}" == "y" ]; then
+				ROMFS_FLASHING_CFG=${OPENOCD_DIR_PATH}/romfs_flashing.cfg
+				if [ "${CONFIG_FS_ROMFS}" = "y" ]; then
+					echo "set romfs_partition_enable 1" > ${ROMFS_FLASHING_CFG}
+				else
+					echo "set romfs_partition_enable 0" > ${ROMFS_FLASHING_CFG}
+				fi
+				pushd ${OS_DIR_PATH}
+				sh ../tools/fs/mkromfsimg.sh
+				if [ ! -f "../build/output/bin/romfs.img" ]; then
+					echo "ROMFS image is not present"
+					exit 1
+				fi
+				popd
+			fi
+
 			# Download all binaries using openocd script
 			pushd ${OPENOCD_DIR_PATH}
-			${OPENOCD_BIN_PATH}/openocd -f artik053.cfg -c ' 	\
+			if [ "${CONFIG_FS_ROMFS}" == "y" ]; then
+				${OPENOCD_BIN_PATH}/openocd -f artik053.cfg -c ' 	\
 				flash_write bl1 ../../bin/bl1.bin; 		\
 				flash_write bl2 ../../bin/bl2.bin; 		\
 				flash_write sssfw ../../bin/sssfw.bin; 		\
 				flash_write wlanfw ../../bin/wlanfw.bin;	\
 				flash_write os ../../../../output/bin/tinyara_head.bin;	\
-			exit'
+				flash_write rom ../../../../output/bin/romfs.img; \
+				exit'
+				rm ${ROMFS_FLASHING_CFG}
+			else
+				${OPENOCD_BIN_PATH}/openocd -f artik053.cfg -c ' 	\
+				flash_write bl1 ../../bin/bl1.bin; 		\
+				flash_write bl2 ../../bin/bl2.bin; 		\
+				flash_write sssfw ../../bin/sssfw.bin; 		\
+				flash_write wlanfw ../../bin/wlanfw.bin;	\
+				flash_write os ../../../../output/bin/tinyara_head.bin;	\
+				exit'
+			fi
```

Comment:
Other than romfs, all the statements in the if and else are common. can you modify to apply condition only to romfs and keep rest all unconditional

Reply:
Chaehee has already tried as u told, but it failed...

Reply:
Dear Mr. Girish, it seems that all reviews are modified and handled properly, therefore if possible, please merge this pull request until today (9/13). :)

Reply:
I think sunghan can merge if all his reviews are addressed